### PR TITLE
Refactor schema loader integration tests

### PR DESCRIPTION
**Pull request by brfrn169**
This PR refactors the schema loader integration tests. I'll add inline comments for the details. Please take a look!

### CODE REVIEW

File: schema-loader/src/integration-test/java/com/scalar/db/schemaloader/SchemaLoaderIntegrationTestBase.java [hunk 1]
```diff
@@ -420,27 +420,8 @@ private void deleteTables_ShouldDeleteTablesWithCoordinator() throws Exception {
...
-            new InputStreamReader(process.getErrorStream(), Charset.defaultCharset()))) {
-      String line;
-      while ((line = input.readLine()) != null) {
-        System.out.println(line);
-      }
-    }
-
-    return process.waitFor();
+  private int executeWithArgs(List<String> args) {
+    return SchemaLoader.mainInternal(args.toArray(new String[0]));
```
Comment (brfrn169) [inline schema-loader/src/integration-test/java/com/scalar/db/schemaloader/SchemaLoaderIntegrationTestBase.java:424]: Previously we run the integration tests with the fat jar, so we need to build the fat jar before running the integration tests. But after this change, we directly call `SchemaLoader.mainInternal()` to run the schema loader, so we no longer need to build the fat jar.

File: server/src/integration-test/java/com/scalar/db/server/SchemaLoaderIntegrationTestWithScalarDbServer.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import java.util.Properties;
```
Comment (brfrn169) [inline server/src/integration-test/java/com/scalar/db/server/SchemaLoaderIntegrationTestWithScalarDbServer.java:6]: Moved the schema loader integration tests with Scalar DB Server to the `server` module.

File: schema-loader/build.gradle [hunk 3]
```diff
@@ -153,90 +137,59 @@ shadowJar {
     mergeServiceFiles()
 }
 
-// The fat jar path
-def schemaLoaderJarPath = "${project.buildDir}/libs/${archivesBaseName}-${project.version}.jar"
-
 task integrationTestCassandra(type: Test) {
-    dependsOn shadowJar
```
Comment (brfrn169) [inline schema-loader/build.gradle:160]: Since the integration tests no longer depend on the fat jar after this change, I removed this dependency.

**Review by Torch3333 — APPROVED** [review]
Nice addition, it will be easier to debug integration test errors. 
LGTM, thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**